src/state/viewer/reducer.ts

```diff
@@ -97,6 +99,14 @@ const actionToConfigMap: TypeToDescriptionMap = {
             isLooping: action.payload,
         }),
     },
+    [SET_EMBED_FULLSCREEN]: {
```

Comment:
can you pull this out into another PR? I have comments/review about this too and I think it would be nicer to get the controls merged and then add the full screen feature